api/migrations/versions/0edb89e87e72_indabaxza_offer_email_templates.py

```diff
@@ -0,0 +1,187 @@
+    language = db.Column(db.String(2), nullable=False)
+    template = db.Column(db.String(), nullable=False)
+    subject = db.Column(db.String(), nullable=False)
+
+    event = db.relationship('Event', foreign_keys=[event_id])
+
+    def __init__(self, key, event_id, subject, template, language):
+        self.key = key
+        self.event_id = event_id
+        self.subject = subject
+        self.template = template
+        self.language = language
+
+
+def upgrade():
+    Base.metadata.bind = op.get_bind()
+    session = orm.Session(bind=Base.metadata.bind)
+    event = session.query(Event).filter_by(key='indabax2023').first()
+
+    offer_award_content = """Dear {title} {firstname} {lastname},
+
+Congratulations! You've been selected for {event_name}!
+
+In addition, you have been awarded a travel grant to attend the IndabaX!
+
+Congratulations, only the very best have been awarded this. Your travel grant has been made possible by our partners at the event, so give them a personal thanks and chat about future opportunities.
+{grants}
+
+The terms and conditions are specified on our website and may be subject to change - https://indabax.co.za/register/travel-grants
+
+Please note that breakfast and lunches are provided but dinner is not, although evening events may have snacks available.
+
+Please follow this link to see details and accept your offer: {host}/offer
+You have up until {expiry_date} to accept the offer, otherwise we will automatically allocate your spot to someone else.
+
+If you are unable to accept the offer for any reason, please do let us know by visiting {host}/offer, clicking "Reject" and filling in the reason. 
+We will read all of these and if there is anything we can do to accommodate you, we may extend you a new offer in a subsequent round.
+
+If you have any queries, please contact us at {event_email_from}
+
+Kind Regards,
+The {event_name} organisers
+"""
+
+    offer_content = """Dear {title} {firstname} {lastname},
+
+Congratulations! You've been selected for {event_name}!
+
+Please note that breakfast and lunches are provided but dinner is not, although evening events may have snacks available.
+
+Please follow the link below to see details and accept your offer: {host}/offer
+You have up until {expiry_date} to accept the offer, otherwise we will automatically allocate your spot to someone else.
+
+If you are unable to accept the offer for any reason, please do let us know by visiting {host}/offer, clicking "Reject" and filling in the reason. 
+We will read all of these and if there is anything we can do to accommodate you, we may extend you a new offer in a subsequent round.
+
+If you have any queries, please contact us at {event_email_from}
+
+Kind Regards,
+The {event_name} organisers
+"""
+    
+    offer_award_template = EmailTemplate('offer-award', event.id, "{event_name} Application Status Update", offer_award_content, 'en')
```

Comment:
This one should be for key `offer-grants`

Reply:
Fixed. The `offer-award` template looked nearly identical unless I missed some small detail.